src/lib/routing-table.js

```diff
@@ -21,14 +21,18 @@ class RoutingTable {
     routes.set(nextHop, route)
   }
 
+  /**
+   * returns true when connectivity is lost
```

Comment:
Should be valid JSDoc, e.g. `@returns {Boolean} True if connectivity has been lost`